containers/python/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM clipper/py-rpc:latest
 MAINTAINER Dan Crankshaw <dscrankshaw@gmail.com>
```

Comment:
Building this no longer works, but I'm also not sure why this file is around in the first place.